Python/pystate.c

```diff
@@ -1388,6 +1389,10 @@ PyThreadState_Next(PyThreadState *tstate) {
 PyObject *
 _PyThread_CurrentFrames(void)
 {
+    // Disable the GC as this can cause a deadlock the interpreter.
+    // See issues 116969 and 106883.
```

Comment:
I think the common way to format something like this is:

```suggestion
    // gh-106883: Disable the GC as this can cause a deadlock the interpreter.
```

Reply:
Done!